src/components/UploadForm/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import axios from "axios";
+import StyledDropzone from "./DropZone";
+export default function UploadInput({ addUrl }) {
+  const uploadFile = (file, signedRequest, url) => {
+    axios
+      .put(signedRequest, file, {
+        headers: {
+          "Content-Type": file.type
+        }
+      })
+      .then(res => {
+        if (res.status === 200) {
+          document.getElementById("preview").src = url;
+          document.getElementById("avatar-url").value = url;
```

Comment:
this should be saved in the state, we have talked about this multiple times, it would be saved in the state and changed in the parent component.